src/init/init.c

```diff
@@ -340,9 +340,8 @@ int sof_main(int argc, char *argv[])
 
 struct device;
 
-static int sof_init(const struct device *dev)
+static int sof_init(void)
 {
-	ARG_UNUSED(dev);
 
```

Comment:
please also remove this empty line - in all chunks

Reply:
done